sequence_processing_pipeline/configuration.json

```diff
@@ -38,6 +38,20 @@
       "job_pool_size": 30,
       "job_max_array_length": 1000
     },
+    "nu-qc": {
+      "nodes": 1,
+      "nprocs": 16,
+      "queue": "qiita",
+      "wallclock_time_in_minutes": 60,
```

Comment:
This is too low, but I don't know what to expect as Qiita is doing IO against panfs

Reply:
@wasade @antgonza Would four hours instead of one be a meaningful change at this point? I would think we'd need to deploy this on qiita-rc and run tests in order to hone in on a good number.